example/spec/s3Access/s3AccessSpec.js

```diff
@@ -142,53 +157,73 @@ describe('When accessing an S3 bucket directly', () => {
       expect(whoami.UserId).toMatch(new RegExp(`.*:${username}`));
     });
 
-    describe('while in the the same region ', () => {
-      it('the bucket contents can be listed', async () => {
-        expect(await canListObjects('us-east-1', creds)).toBe('true');
-      });
 
-      it('the data can be downloaded', async () => {
-        expect(await canGetObject('us-east-1', creds)).toBe('true');
-      });
+    function executeTestsAgainst(testBucket) {
+      describe('while in the the same region ', () => {
+        it('the bucket contents can be listed', async () => {
+          expect(await canListObjects('us-east-1', testBucket, creds)).toBe('true');
+        });
 
-      it('a write is rejected', async () => {
-        expect(await canWriteObject('us-east-1', creds)).toBe('false');
-      });
-    });
+        it('the data can be downloaded', async () => {
+          expect(await canGetObject('us-east-1', testBucket, creds)).toBe('true');
+        });
 
-    describe('while outside the region ', () => {
-      it('the bucket contents can NOT be listed', async () => {
-        expect(await canListObjects('us-west-2', creds)).toBe('false');
+        it('a write is rejected', async () => {
+          expect(await canWriteObject('us-east-1', testBucket, creds)).toBe('false');
+        });
       });
 
-      it('the data can NOT be downloaded', async () => {
-        expect(await canGetObject('us-west-2', creds)).toBe('false');
-      });
+      describe('while outside the region ', () => {
+        it('the bucket contents can NOT be listed', async () => {
+          expect(await canListObjects('us-west-2', testBucket, creds)).toBe('false');
+        });
 
-      it('a write is rejected', async () => {
-        expect(await canWriteObject('us-east-1', creds)).toBe('false');
-      });
-    });
-  });
+        it('the data can NOT be downloaded', async () => {
+          expect(await canGetObject('us-west-2', testBucket, creds)).toBe('false');
+        });
 
-  describe('with third-party/invalid credentials', () => {
-    const thirdPartyCredentials = {
-      accessKeyId: 'AKIAIOSFODNN7EXAMPLE',
-      secretAccessKey: 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY',
-      sessionToken: 'FAKETOKENdfkjaf9rufjfdklajf',
-      expiration: '2019-02-26 00:08:18+00:00'
-    };
+        it('a write is rejected', async () => {
+          expect(await canWriteObject('us-east-1', testBucket, creds)).toBe('false');
```

Comment:
So a write should be rejected whether you are in-region or not? Even still, it seems like this test should be updated to use `us-west-2` otherwise it's just a repeat of the test on line 171. Or it could be removed.

Reply:
Good catch, I think that was a bug before.  it should be 'us-west-2'.